src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -120,6 +120,15 @@
       <Context>ProjectSubscriptionService;BrowseObject</Context>
     </PropertyPageSchema>
 
+    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)AnalyzerReference.xaml">
+      <Context>Project;ProjectSubscriptionService;BrowseObject</Context>
```

Comment:
If ProjectSubscriptionService is not added, then ProjectSubscription_Changed events don't fire for the analyzerreferneces. There's another place in the code where we look for the rule in the Project catalog. I noticed that AssemblyReference.xaml (for eg) does get published to both Project and ProjectSubscriptionService (as in I see them in the catalog under the debugger) but they are not published by us. I assume CPS does this somehow. We need to figure out why.